src/components/StartGame.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "../App.css";
+import { useRecoilState } from "recoil";
+import Box from '@mui/material/Box';
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import { gameStartedState, isPlayingState } from "../state";
+
+
+function StartGame(){
+  const [, setGameStarted] = useRecoilState(gameStartedState);
+  const [, setIsPlaying] = useRecoilState(isPlayingState);
```

Comment:
Can update these to use https://recoiljs.org/docs/api-reference/core/useSetRecoilState
e.g.
```
  const setGameStarted = useSetRecoilState(gameStartedState);
```